pandas/tests/plotting/test_converter.py

```diff
@@ -108,6 +110,7 @@ def test_option_no_warning(self):
         register_matplotlib_converters()
         with ctx:
             ax.plot(s.index, s.values)
+        plt.close()
```

Comment:
is this a typo? `unit`?

Reply:
ahh, sorry!! just noticed codes were folded, i thought this was related to your change above.